playbooks/roles/cluster/tasks/workflow.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Configure Openshift authenticate job template
+  tower_job_template:
+    name: "{{ cluster_job_template_openshift_auth_name }}"
+    description: "{{ cluster_job_template_openshift_auth_desc }}"
+    job_type: "{{ cluster_job_template_openshift_auth_type }}"
+    playbook: "{{ cluster_job_template_openshift_auth_playbook }}"
+    project: "{{ cluster_job_template_openshift_auth_project }}"
+    credential: "{{ tower_credential_bundle_default_name }}"
+    extra_vars_path: "files/openshift_tower_auth.yml"
```

Comment:
Will you take another look at the authentication job created in the tower boot strap process. I don't like this extra vars taking in credentials, as it ends up with the decrypted vars in the UI. Best to read them in in the group vars, vaulted up.

Reply:
Cool, switched them out and I'll PR the group var to the other repo.